.github/workflows/tests.yml

```diff
@@ -39,8 +48,7 @@ jobs:
           python -m build --sdist
 
       - name: Run mypy
-        run: python -m mypy confection
-        if: matrix.python_version != '3.6'
+        run: python -m mypy confection/__init__.py confection/util.py
```

Comment:
I think it would be better to use `# mypy: ignore` in the relevant parts of the test suite instead? I think we'll have trouble remembering a custom mypy usage in future development?